docker/Dockerfile.pdo-dev

```diff
@@ -87,7 +87,6 @@ RUN apt-get update \
     libsecp256k1-dev \
     libssl-dev \
     libtool \
-    tinyscheme \
```

Comment:
I'm wondering if there's any dependency here due to tinyscheme -- difficult to figure it out without trying to remove any of them.

Reply:
Not entirely sure I understand the comment. But tinyscheme has no other dependencies.